Modules/Quotes/Filament/Company/Resources/Quotes/QuoteResource.php

```
@@ -60,4 +62,29 @@ public static function getPages(): array
+
+    public static function canViewAny(): bool
+    {
+        return auth()->user()?->can(Permission::VIEW_QUOTES->value) ?? false;
+    }
+
+    public static function canCreate(): bool
+    {
+        return auth()->user()?->can(Permission::CREATE_QUOTES->value) ?? false;
+    }
+
+    public static function canView(Model $record): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$record'. (undefined)

(UnusedFormalParameter)